code/game/objects/items/trader.dm

```diff
@@ -24,6 +25,7 @@
 /obj/item/weapon/storage/bluespace_crystal
 	name = "Natural bluespace crystals box"
```

Comment:
Makes me want to depreciate the whole damn file and also pick a better sprite than the damn uplink syndie box..